src/Microsoft.TestPlatform.ObjectModel/Navigation/DiaSession.cs

```diff
@@ -47,7 +47,7 @@ public DiaSession(string binaryPath)
         /// search path.
         /// </param>
         public DiaSession(string binaryPath, string searchPath)
-            : this(binaryPath, searchPath, GetSymbolReader(binaryPath))
+            : this(binaryPath, searchPath, GetSymbolReader(binaryPath, !Path.IsPathRooted(binaryPath) ? Directory.GetCurrentDirectory() : null))
```

Comment:
Can we use IFileHelper API here.

Reply:
Directory.GetCurrentDirectory() might not give the right path. 
https://github.com/dotnet/project-system/issues/2239